src/common/AndroidUtils.ts

```diff
@@ -350,12 +344,25 @@ export class AndroidSDKUtils {
                         }, 3000);
                     });
                     child.stdout.on('exit', () => resolve(true));
-                    child.stderr.on('error', () => reject(false));
+                    child.stderr.on('error', (err) => {
+                        reject(
+                            new Error(
+                                `Could not create emulator. Command failed: ${createAvdCommand}\n${err}`
+                            )
+                        );
+                    });
+                    child.stderr.on('data', (data) => {
+                        reject(
```

Comment:
depending on the type of errors, sometimes they are invoked on stderr.on('data') 🤷‍♂️